src/LetterChanger.java

```diff
@@ -1,57 +1,61 @@
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Scanner;
 
 class LetterChanger {
     private static Scanner scanner = new Scanner(System.in);
 
-    public static void main(String[] args) {
-        int numberOfWords = getNumberOfWords();
-        String[] words = readWords(numberOfWords);
-        String change = readChangeString();
-        if (change == "upper") {
-            changeWordsToUpperCase(words);
-        } else if(change == "lower") {
-            changeWordsToLowerCase(words);
-        } else {
-            System.out.println("Nieprawidłowy tekst, zostawiam oryginalny tekst");
-        }
+    private final String UPPER_CASE = "upper";
+    private final String LOWER_CASE = "lower";
+
+    private List<String> words = new ArrayList<>();
 
-        printArray(words);
+    public void addAndPrintWords() {
+        addWords(getNumberOfWords());
+        chooseOption();
     }
 
-    private static String readChangeString() {
+    private void chooseOption() {
```

Comment:
I would name this method transformText()

Reply:
-,-